src/resources/views/fields/table.blade.php

```diff
@@ -140,13 +140,13 @@
                         }
                     }
 
-                    if( typeof $scope.items != 'undefined' && $scope.items.length ){
+                    if( typeof $scope.items != 'undefined' ){
```

Comment:
Yeah parts of the code could be more concise, but I figured that was out of scope for this PR.